plaid/client.py

```diff
@@ -310,17 +312,19 @@ def connect_step(self, url, *args, **kwargs):
         return self._step(url, 'POST', *args, **kwargs)
 
     @inject_url('connect')
-    def connect_update(self, url, login):
+    def connect_update(self, url, login=None, options=None):
         '''
-        Update a user who has already connected
+        Update a user who has already connected.
 
         `login`         dict
             `username`        str     username for the bank account
             `password`        str     The password for the bank account
             `pin`             int     (optional) pin for the bank account
-
+        `options`       dict
```

Comment:
Note that options is supposed to align with `login`, not the children of `login`. While looking at this I noticed that the descriptions for the children of options were off by a space, so I fixed those. I'm not sure if that's what you were pointing out or not.